Dockerfile

```diff
@@ -11,9 +11,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Specify the version of Chrome that matches the version of chromedriver in the package.json.
-# A list of Chrome versions can be found here: 
+# A list of Chrome versions can be found here:
 # https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable
-ARG CHROME_VERSION=91.0.4472.114-1
+ARG CHROME_VERSION=101.0.4951.64-1
```

Comment:
If we're going to change this here, we should change it everywhere we reference chrome/chromedriver.

Reply:
This should probably be updated in a separate PR (necessary for Jenkins to run) - but I'm also unsure of the best way to find all those.

Reply:
removed from this PR